index.html

```diff
@@ -4,16 +4,27 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./styles/styles.css" />
+    <link rel="stylesheet" href="styles.css" />
     <title>Quiz App</title>
   </head>
   <body>
     <header>QuizApp for WebDevs</header>
     <main>
       <section class="card">
-        <p class="question">What property flips the axes in flexbox?</p>
-        <button class="button-answer">Show answer</button>
-        <button class="button-bookmark"></button>
+        <p class="card__question">What property flips the axes in flexbox?</p>
+        <button class="card__button-answer">Show answer</button>
+        <button class="card__button-bookmark"></button>
```

Comment:
I was wondering what would be a good element for the bookmark, not sure if button is the best option or maybe something else. I think I used the "i"-element because i inserted an icon but also not sure if this is better.